### DISCUSSION TITLE
Was the onFilter event removed from the suggestions utility?

### DISCUSSION THREAD

**Discussion by deRaaf** [ueberdosis/tiptap#5690] [General Questions]
I'm trying to implement a variant of the slash command experiment based on [this](https://github.com/ueberdosis/tiptap/pull/757#issuecomment-671053224) implementation: 

```
    onFilter: (items, query) => {
        return items.filter((item) => item.name.toLowerCase().startsWith(query.toLowerCase())).slice(0, 10);
    }
```

But it's not working and there's nothing in the docs.

**Comment by guanriyue** [comment]
It seems like you're looking at the v1 API. In Tiptap v2, you can retrieve the query keyword directly within `suggestion.render.onUpdate`.

https://github.com/ueberdosis/tiptap/blob/88371561bbb564c9c94a6ebcfdc44fbf0fbeeb25/packages/suggestion/src/suggestion.ts#L88-L95
